specification/purview/Azure.Analytics.Purview.DataMap/tspconfig.yaml

```
@@ -0,0 +1,36 @@
+parameters:  
+  "service-dir":
+    default: "sdk/azure.analytics.purview.datamap"
+  "dependencies":
+    "additionalDirectories": []
+    default: ""
+emit:
+  - "@azure-tools/typespec-autorest"
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/.."
+    examples-directory: "{project-root}/examples"
+    output-file: "{azure-resource-provider-folder}/Azure.Analytics.Purview.DataMap/{version-status}/{version}/openapi.json"
+  "@azure-tools/typespec-python":    
+    package-dir: "azure-analytics-purview-datamap"
+    package-name: "{package-dir}"  
+    package-mode: dataplane
+  "@azure-tools/typespec-csharp":
+    package-dir: "Azure.Analytics.Purview.DataMap"
+    namespace: "{package-dir}"
+    clear-output-folder: true
+    model-namespace: false
+  "@azure-tools/typespec-ts":
+    package-dir: "azure-analytics-purview-datamap-rest"
```

Usually we won't duplicate the `azure-` prefix in sub folder.
```suggestion
    package-dir: "analytics-purview-datamap-rest"
```

---

Thanks. Updated. Is it also apply to other languages as well?

---

I don't think so, Java would require `azure-` AFAIK.

---

@yifan-zhou922
Basically package-dir is the thing that you would review in SDK review meeting. So you should be more clear about this than us...